content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md

```diff
@@ -101,7 +101,7 @@ be done by the egress gateway, as opposed to by the sidecar in the previous exam
 
     {{< tab name="mutual TLS enabled" cookie-value="enabled" >}}
 
-    {{< text bash >}}
+    {{< text_hack bash >}}
```

Comment:
There are 2 more places that need the hack:

https://deploy-preview-3928--preliminary-istio.netlify.com/docs/examples/advanced-gateways/egress-gateway/#egress-gateway-for-http-traffic

https://deploy-preview-3928--preliminary-istio.netlify.com/docs/examples/advanced-gateways/wildcard-egress-hosts/#configure-traffic-through-egress-gateway-with-sni-proxy

Reply:
More places:
https://deploy-preview-3928--preliminary-istio.netlify.com/blog/2018/egress-mongo/#direct-tls-egress-traffic-through-an-egress-gateway
https://deploy-preview-3928--preliminary-istio.netlify.com/docs/examples/advanced-gateways/egress-gateway/#egress-gateway-for-https-traffic